README.md

```diff
@@ -140,137 +209,244 @@ Let's have a look on the Flume event headers:
 * The time-to-live (or <b>ttl</b>) specifies the number of re-injection retries in the channel when something goes wrong while persisting the data. This re-injection mechanism is part of the reliability features of Flume. -1 means inifinite retries.
 * The <b>destination</b> headers is used to identify the persistence element within the used storage, i.e. a file in HDFS, a MySQL table or a CKAN resource. This is added by a custom interceptor called `DestinationExtractor` added to the Flume's suite. See the <i>doc/design/interceptors</i> document for more details.
 
-Finally, the channel is a simple MemoryChannel behaving as a FIFO queue, and from where the different sinks extract the events in order to persist them; let's see how:
+The body simply contains a byte representation of the HTTP payload that will be parsed by the sinks.
 
 [Top](#top)
 
-###<a name="section3.1"></a>OrionHDFSSink persistence
-
-This sink persists the data in files, one per each entity, following this entity descriptor format:
+###<a name="section3.4"></a>Data persistence
+####<a name="section3.4.1"></a>HDFS persistence
```

Comment:
It is already done. What appears in these sections is an "extract" of what is written there. The purpose for these sections is just to show how the data of the proposed example is persisted within the different storages; this and many other information is also written in the specific sink document.